static/css/event.less

```diff
@@ -0,0 +1,210 @@
+/* ==========================================================================
+   cfgov-refresh
+   Page-specific styles for event post detail page.
+   ========================================================================== */
```

Comment:
This file could use some org/comment love but I didn't have time in the last clean up